client/gefyra/types.py

```diff
@@ -311,6 +311,14 @@ class GefyraInstallOptions:
             help="The storage size for the Stowaway PVC in Mi (default: 64)",
         ),
     )
+    stowaway_max_connection_age: int | None = field(
+        default_factory=lambda: None,
+        metadata=dict(
+            help=(
+                "The maximum age of a Stowaway connection in seconds (default: None)"
```

Comment:
I'd suggest naming it "maximum client connect age" and leaving out "stowaway" as a term. It would address the nature of the effect more than the technical aspect of the implementation.